modules/EvseV2G/EvseV2G.cpp

```diff
@@ -16,11 +17,13 @@ void log_handler(openssl::log_level_t level, const std::string& str) {
         // ignore debug logs
         break;
     case openssl::log_level_t::warning:
-        EVLOG_warning << str;
+        // avoid logging function and line number
+        BOOST_LOG_SEV(::global_logger::get(), ::Everest::Logging::warning) << str;
```

Comment:
Is this really neccessary? We may want to replace boost::log at some point, so the EVLOG_nn abstraction should be used I think. Maybe we can remove the funtion and line number from all warning logs instead as it makes logs quite unreadable

Reply:
I don't have a good solution.
We should use a consistent logging approach.
I was asked to make the TLS server logging configurable rather than use the standard approach.

EVLOG_warning etc. all have a memory leak which makes use of valgrind/memcheck problematic (especially for unit tests).

With EVLOG_warning ... they will always show EvseV2G.cpp and that line number despite the error occurring elsewhere (staging/tls ...). 

The file & line number have been very useful in logs in the past to help track down an issue. But showing the wrong file and line isn't useful.

I will revert to EVLOG_warning << str; and EVLOG_error << str; since that is the least worst option.

Reply:
Maybe we can change it such that info and warning don't output line numbers. Critical and error maybe still do. It would be nice to have a warning level that is not as verbose as it is now

Reply:
added a note to liblog so that removing file & line can be considered.
https://github.com/EVerest/liblog/issues/22